src/robotide/log/log.py

```diff
@@ -20,10 +20,13 @@
 import atexit
 import glob
 import sys
+import io
 
 from robotide.pluginapi import Plugin, ActionInfo, RideLog
 from robotide import widgets
 from robotide import context
+from robotide.context import IS_WINDOWS
```

Comment:
this isn't used?

Reply:
Correct. I was trying other options in Windows.